webapp/javascript/components/ComparisonApp.jsx

```diff
@@ -24,6 +24,51 @@ function ComparisonApp(props) {
     props;
   const { rawLeft, rawRight } = comparison;
 
+  const [linkedSearch, setLinkedSearch] = useState({
+    isSearchLinked: false,
+    linkedSearchQuery: '',
+    resetLinkedSearchSide: '',
+  });
+
+  const setSearchQuery = (query) => {
+    setLinkedSearch((x) => {
+      return {
+        ...x,
+        linkedSearchQuery: query,
+      };
+    });
+  };
+  const toggleLinkedSearch = (side) => {
+    const { isSearchLinked } = linkedSearch;
+    if (isSearchLinked) {
```

Comment:
Can we return early here if `isSearchLinked` is false? The nesting here doesn't add much value IMO.

Reply:
done.